### PR TITLE
Add fishway content.

### PR DESCRIPTION
**Pull request by lliss**
Connects #160 

This adds the fishway zone. See screenshot for location.

screencap

There are a few issues with this.

 1. I was unable to find the intro picture that goes with this. Each deck needs to include an alignment photo. I found a picture of the fishway but it surely isn't the view you would see if you were at that location.
 2. Video is very short. The video is only about 2 seconds. It's a bit jarring. 
 3. Location of fishway zone overlaps with the decks zone (see screenshot).

### CODE REVIEW

**Comment by arobbi7** [comment]
I am following up with Victoria about these issues.

**Comment by mmcfarland** [comment]
Re: 3, making them smaller seems reasonable, the current radius make a bit of the circle over an inaccessible portion of the water anyway.  We'll see what Victoria says.

**Comment by lliss** [comment]
We do have the ability to make each one smaller independently of the others.

**Comment by arobbi7** [comment]
New info:
1. Picture should be coming in this afternoon, I'll put it in the fileshare.
2. Substitute the quillback clip for the short shad one.
3. Victoria suggests that we move the decks zone (RII) toward the east along the pier a bit to make room for the Fishway zone. If there's still not enough room we should probably reduce the size of both zones so they fit without overlapping.

**Comment by arobbi7** [comment]
Picture is now on the fileshare smb://fileshare/projects/PWD_StormwaterBilling/downloads/Water Revealed/2015_04_13_DropboxDownload/Images/Fishway_photo.jpg

**Comment by lliss** [comment]
The new media looks a lot better. I moved the zones slightly so they shouldn't overlap anymore. This is now ready for another test.

**Comment by arobbi7** [comment]
This looks great!

File: src/quests.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
                     "text": "By the 1840s the chaste, neoclassical architecture of the buildings, the landscaped garden, and the promenades up to and around the reservoirs on \"Fair Mount,\" as the rise was identified as early as 1687 on a map of Philadelphia, were major attractions. Ornamental sculptures, fountains, and airy gazebos added to the visual pleasure of the site.", 
                     "attribution": "Jane Mork Gibson"
                 }, 
-                "radius": 5,
```
Comment (mmcfarland) [inline src/quests.json]: Not precisely relevant to this PR, but I think we should .gitignore this file since it's generated from the grunt command.

**Comment by mmcfarland** [comment]
+1 Looks good to me too

**Pull request merged**